struqture-py/src/spins/decoherence_product.rs

```diff
@@ -35,8 +35,8 @@ use struqture_py_macros::{mappings, product_wrapper};
 ///
 /// `DecoherenceProduct().x(0).z(2)`.
 ///
-/// DecoherenceProduct is  supposed to be used as input for the function `add_noise`,
-/// for instance in the spin system classes QubitLindbladOpenSystem, SpinLindbladNoiseSystem or QubitLindbladNoiseOperator,
+/// DecoherenceProduct is supposed to be used as input for the function `add_noise`,
+/// for instance in the spin system classes PauliLindbladOpenSystem or PauliLindbladNoiseOperator,
```

Comment:
Is it correct that here previously 3 classes were mentioned but now only two are?

Reply:
Yes, the system layer has been removed